### PR TITLE
Added connection verification and retry if is not connected

### PR DESCRIPTION
**Pull request by alexjavabraz**

### CODE REVIEW

File: src/services/WalletService.ts
Comment (josedahlquist) [inline src/services/WalletService.ts:139]: we should probably raise an exception if we cannot reconnect

File: src/services/LedgerService.ts
Comment (josedahlquist) [inline src/services/LedgerService.ts:27]: how can we guarantee the ledger is connected now?
Should we call isConnected from here after we get a newer instance?

**Review by ronaldsg20 — APPROVED** [review]
LGTM

File: src/services/LedgerService.ts
Comment (alexjavabraz) [reply src/services/LedgerService.ts:27]: To avoid a loop, we will verify if is connected, if is not, we will try to reconnect once, and then, try to obtain the wallet's data, if we can't connect, the app will raise an exception and the application will show a popup to the user.

**Review by josedahlquist — APPROVED** [review]

**Comment by josedahlquist** [comment]
approved but will merge after pegout-status is merged

**Pull request merged**
